tictactoe/adapter-script-generation/src/main/kotlin/com/anaplan/engineering/azuki/tictactoe/adapter/scriptgen/TicTacToeRunnableScenarioClassGenerator.kt

```diff
@@ -0,0 +1,39 @@
+package com.anaplan.engineering.azuki.tictactoe.adapter.scriptgen
+
+import com.anaplan.engineering.azuki.core.parser.ScenarioParsingContext
+import java.util.*
+
+object TicTacToeRunnableScenarioClassGenerator {
+
+    // TODO - format class
+    fun generate(
+        className: String = "Generated_" + UUID.randomUUID().toString().replace("-", "_"),
+        packageName: String? = null,
+        implementationVersions: Map<String, String> = emptyMap(),
+        scenarioScript: String,
+    ) = RunnableScenarioClass(className, packageName, """
+package ${packageName ?: ""}
```

Comment:
⛏️ I don't see why you would allow a null `packageName` and default it to an empty string here, instead of just having a default value of `""`.

Either way, though, this seems odd... can you actually have an empty package directive? I would rather omit the directive entirely if no name is supplied:
```
${if (packageName.isEmpty()) "" else "package $packageName"}
```